utils/docker/orakwlum-frontend/run_frontend.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+echo "Replaced Host Name with [$1]"
+server_name="$1"
+sed -i 's/$HOST_NAME/'$server_name'/g' /etc/nginx/sites-enabled/*
```

Comment:
It's **$HOST** instead of **$HOST_NAME**, see https://github.com/gisce/orakwlum-api-pub/blob/models_refactoring/utils/nginx_vhost_conf/production/orakwlum.conf#L7